composer.json

```diff
@@ -9,11 +9,11 @@
     ],
     "license": "Apache-2.0",
     "keywords": ["Drupal", "AMP", "mobile"],
-    "homepage": "https://github.com/Lullabot/amp-library",
+    "homepage": "https://github.com/ewilde-imperial/amp-library",
     "require" : {
         "php" : ">=5.5",
         "querypath/querypath": ">=3.0.4",
-        "sebastian/diff": "^1.2 || ^2 || ^3",
+        "sebastian/diff": "^4",
```

Comment:
is it really required to be ^4, can' t we just also also 4? doesn't look like there is a code changed necessary for this.